features/support/api_helpers.rb

```diff
@@ -211,6 +211,12 @@ def create_live_service(framework_slug, lot_slug, supplier_id)
       puts 'Lot slug not recoginsed'
   end
 
+  if lot_slug == 'digital-specialists' and role
+    service_data['services']["#{role}Locations".to_sym] = service_data['services'].delete(:developerLocations)
```

Comment:
Yes you are entirely correct. Good shout on making this more readable, I will add some comments.

We need to delete the developer role so when we apply for a developer brief we are on the specialists lot but do not have the specialists role (i.e. we just have designer and not developer). This is one of the eligibility tests that I have added.